Please fix!

/home/vsts/work/1/s/src/example.cpp:12:29: error: ‘printf’ was not declared in this scope
     printf("Vyacheslavovich");
                             ^

--- a/src/example.cpp
+++ b/src/example.cpp
@@ -1,5 +1,5 @@
 #include "example.h"
-#include <cstdlib>
+#include <iostream>
 
 
 double add_numbers(const double f1, const double f2)

--- a/src/example.cpp
+++ b/src/example.cpp
@@ -1,15 +1,17 @@
 #include "example.h"
+#include <cstdlib>
+
 
 double add_numbers(const double f1, const double f2)
 {
     return f1 + f2;
-    printf("Du hast");
+    std::cout <<"Du hast ";
 }
 
 double subtract_numbers(const double f1, const double f2)
 {
     return f1 - f2;
-    printf("Vyacheslavovich");
+    std::cout <<"Vyacheslavovich";
 }
 
 double multiply_numbers(const double f1, const double f2)

